src/components/Distributive.jsx

```diff
@@ -0,0 +1,70 @@
+// Importing `useState` and `useEffect` hooks from "react" library
+import { useState, useEffect } from "react";
+// Importing `PostMessage` and `MessageList` components from respective files
+import { PostMessage } from "./PostMessage";
+import { MessageList } from "./MessageList";
+import './Distributive.css'
+
+
+// Declaring a functional component named `Distributive`
+export const Distributive = () => {
+
+    // Declaring state `loading` and its updater function `setLoading`, initializing it with `false`
+    const [loading, setLoading] = useState(false);
```

Comment:
You can set the initial state as true because when the app renders it will be true per default.